src/Microsoft.HttpRepl/Commands/ConnectCommand.cs

```diff
@@ -109,6 +109,8 @@ private static void WriteStatus(IShellState shellState, HttpState programState)
             {
                 shellState.ConsoleManager.WriteLine(string.Format(Resources.Strings.ConnectCommand_Status_Swagger, programState.SwaggerEndpoint));
             }
+
+            shellState.ConsoleManager.WriteLine(Resources.Strings.HelpCommand_Core_Details_Line2.Bold().Cyan());
```

Comment:
Since the resource string name isn't very helpful at identifying what's being written, I'd add a comment saying what we're always writing to the console.

```suggestion
            // Always show help link after connecting
            shellState.ConsoleManager.WriteLine(Resources.Strings.HelpCommand_Core_Details_Line2.Bold().Cyan());
```